Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:16
```

Comment:
I think it would be better to use Node 20, since that's the latest stable release